packages/rendermime/style/base.css

```diff
@@ -613,3 +613,15 @@ h6:hover .jp-InternalAnchorLink {
 .jp-RenderedHTMLCommon > *:last-child {
   margin-bottom: 0.5em;
 }
+
+.jp-mimeType-highlight {
+  background-color: var(
+    --jp-search-unselected-match-background-color
+  ) !important;
+  color: var(--jp-search-unselected-match-color) !important;
+}
+
+.jp-mod-selected.jp-mimeType-highlight {
+  background-color: var(--jp-search-selected-match-background-color) !important;
+  color: var(--jp-search-selected-match-color) !important;
+}
```

Comment:
@fcollonval it seems to me that `.jp-mimeType-highlight` is not used, is this correct or am I missing something?

Reply:
This is correct - this is an artifact of my trial for searching the unrendered output.